src/styles/less/MatchUp.less

```diff
@@ -185,3 +185,27 @@
   height: 11rem;
   width: 10rem;
  }
+
+ // Styling for "Go to Point Share" button on RenderMatchUp.js
+.to-pointshare-btn {
+  @media @tablet {
+    width: 30%;
+  }
+}
+.to-pointshare-btn {
+  @media @tablet {
+    display: none;
+  }
+  @media @mobile {
+    display: none;
+  }
+}
+.to-pointshare-btn {
```

Comment:
I am curious why 190 to 202 was not consolidated into 203? And why lines 191 and 196 are not in the same object? Everything looks great though.

Reply:
I agree with these critiques. Looks like some of this can be moved around and will have a much better flow to everything. Much more concise and easier to read.

Reply:
Thank you for bringing that to my attention @xpinero and @StnsGeneral. I've consolidated the code to be contained within the same button class styling. Please review and let me know if this is easier to read/more concise. I am open to any other feedback you may find, too.